code/go/0chain.net/smartcontract/zcnsc/stakepool.go

```diff
@@ -0,0 +1,292 @@
+package zcnsc
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+
+	chainstate "0chain.net/chaincore/chain/state"
+	cstate "0chain.net/chaincore/chain/state"
```

Comment:
`chainstate` and `cstate` are the same package, so use cstate perhaps.

Reply:
Fixed